myst_nbs/pymc3_howto/api_quickstart.myst.md

```diff
@@ -84,13 +87,16 @@ Every probabilistic program consists of observed and unobserved Random Variables
 help(pm.Normal)
 ```
 
-In the PyMC3 module, the structure for probability distributions looks like this:
+In the PyMC module, the structure for probability distributions looks like this:
 
-[pymc3.distributions](../api/distributions.rst)
-- [continuous](../api/distributions/continuous.rst)
-- [discrete](../api/distributions/discrete.rst)
-- [timeseries](../api/distributions/timeseries.rst)
-- [mixture](../api/distributions/mixture.rst)
+{ref}`_api_distributions`
+- {ref}`Continuous`
+- {ref}`Discrete`
+- {ref}`Multivariate`
+- {ref}`Mixture`
+- {ref}`Timeseries`
+- {ref}`Censored`
+- {ref}`Simulator`
```

Comment:
These don't have any target defined, we'd need to add them in pymc versioned docs so we can reference them here